.babelrc

```diff
@@ -0,0 +1,8 @@
+{
+  "presets": ["react", "stage-1", "es2015"],
```

Comment:
Babel 6 doesn't do anything out of the box now. You define sets of plugins (transformers) to enable on your code. A group of these plugins is called a preset. Here we're using `react` (enables jsx compilation), `stage-1` (stage 1 ES proposals), and `es2015` (all the ES2015 goodies).